charts/home-assistant/templates/deployment.yaml

```diff
@@ -332,7 +332,7 @@ spec:
           imagePullPolicy: {{ .Values.appdaemon.image.pullPolicy }}
           ports:
             - name: appdaemon
-              containerPort: {{ .Values.appdaemon.service.port }}
+              containerPort: 5050
               protocol: TCP
           env:
             - name: HA_URL
```

Comment:
You can hardcode the line below this to `value: "http://localhost:8123"` ?

Reply:
There's actually other places in this file that are referencing those ports too e.g.

```
            - name: HA_URL
              value: "http://localhost:{{ .Values.service.port }}"
            {{- if .Values.appdaemon.ingress.enabled }}
            - name: DASH_URL
              #value: http{{ if .Values.appdaemon.ingress.tls }}s{{ end }}://{{ index .Values.appdaemon.ingress.hosts 0 }}
              value: http://0.0.0.0:{{ .Values.appdaemon.service.port }}
```

Reply:
Good catch. That will need to be `http://serviceName:8123`. I'll clean that up and check the others.